model/src/application/interaction/application_command/mod.rs

```diff
@@ -26,6 +26,11 @@ use serde::Serialize;
 #[derive(Clone, Debug, Eq, PartialEq, Serialize)]
 #[serde(rename(serialize = "Interaction"))]
 pub struct ApplicationCommand {
+    /// Permissions the app or bot has within the channel the
+    /// interaction was sent from. It is none if the interaction
+    /// happens in a direct message channel.
```

Comment:
doc lines are wrapped to 80, and we usually have the first sentence on its own line
```suggestion
    /// App's permissions in the channel the interaction was sent from.
    /// 
    /// None if the interaction happens in a direct message channel.
```